RAKE/RAKE.py

```diff
@@ -153,13 +153,13 @@ def __init__(self, stop_words, regex='[\W\n]+', minCharacters=1, maxWords=5, min
         self.__minFrequency = minFrequency
 
     def run(self, text):
-        sentence_list = split_sentences(text)
+        sentence_list = split_sentences(text, regex='[\W\n]+', minCharacters=1, maxWords=5, minFrequency=1)
```

Comment:
These defaults need to be moved to the run method definition on line 155, or they will be unchangeable by users.